token-holders/staking-rules.md

```diff
@@ -0,0 +1,12 @@
+These are the current rules of Pyth staking:
+
+1. Pyth staking is organized in one-week epochs. Each Thursday at 00:00am UTC (midnight between Wednesday and Thursday), a new epoch starts. 
+2. When a user stakes their Pyth tokens, their stake will only become active at the start of the next epoch.   
+3. When a user requests to unlock their Pyth tokens, the tokens will go into cooldown at the end of the current epoch and will be withdrawable after one full epoch.
+
+Examples: 
+- Daniel requests to lock his tokens in epoch 1, his tokens will become locked on epoch 2.
```

Comment:
Daniel requests to lock his tokens in the middle of epoch 1. His tokens will become locked at the start of epoch 2 (after about three days in this case).